cli/cmd/vuln_html.go

```diff
@@ -186,7 +186,7 @@ func calcHtmlBarChartWidth(severity string, htmlData vulnImageAssessmentHtml) st
 	case "info":
 		return fmt.Sprintf("%f%%", htmlData.InfoVulnPercent())
 	default:
-		return fmt.Sprintf("0")
+		return "0"
```

Comment:
@afiune this lint failed for me locally, so I went and fixed here if that's cool...

Reply:
Awesome! I wonder why we didn't catch this before.... 🤔

# Thank you
![tenor-240583896](https://user-images.githubusercontent.com/5712253/113461725-fbb48f00-93da-11eb-9869-f5170ab68368.gif)

Reply:
@afiune; Perhaps this is a difference in `golangci-lint` versions.  brew gave me `golangci-lint/1.39.0`...